### ISSUE TITLE
Accessibility warning when opening dialogs

### ISSUE THREAD

**Issue by sguhl** [b310-digital/teammapper#977]
**Describe the bug**
An Accessibility warning appears when opening a modal.

**To Reproduce**
1. Create a map
2. Open a modal (e.g. the info modal using the info button in the top right corner)

**Expected behavior**
- No accessibility warnings should appear.
- When a modal is open, the background should be properly hidden from screen readers and the focus should be moved to the modal.

**Screenshots**

<img width="602" height="168" alt="Image" src="https://github.com/user-attachments/assets/bc4e7fbe-b616-473d-9694-32d93dfa1eee" />

**Comment by sguhl** [comment]
I can't reproduce the Warning anymore. I found this [PR](https://github.com/angular/components/pull/31030) in the angular repo that should have resolved this. I can't find it in the Changelogs, but I think it's probably included in one of the recent angular updates.
